exporters/metric/datadog/example_test.go

```diff
@@ -98,7 +98,7 @@ func ExampleExporter() {
 	}
 
 	// Output:
-	// myrecorder.max:100|g|#env:dev,l:1
+	// myrecorder.max:100|g|#env:dev,l:1,service.name:unknown_service:datadog.test,telemetry.sdk.language:go,telemetry.sdk.name:opentelemetry,telemetry.sdk.version:0.18.0
```

Comment:
This is going to be tricky. Every upgrade this test will need to be updated.

Reply:
Yeah, though if it's just the version identifier we might be able to work that into `pre_release.sh` and automate it.